lib/internal/modules/esm/worker.js

```diff
@@ -0,0 +1,25 @@
+const { workerData } = require('internal/worker_threads');
+const { ESMLoader } = require('internal/modules/esm/loader');
+
+const { comsChannel } = workerData;
```

Comment:
```suggestion
const { commsChannel } = workerData;
```

And other references.